code/modules/jobs/job_types/roguetown/church/templar.dm

```diff
@@ -239,26 +248,49 @@
 		H.mind.adjust_skillrank(/datum/skill/misc/reading, 3, TRUE)
 		H.mind.adjust_skillrank(/datum/skill/magic/holy, 3, TRUE)
 		H.mind.adjust_skillrank(/datum/skill/misc/medicine, 1, TRUE)
-		ADD_TRAIT(H, TRAIT_RITUALIST, TRAIT_GENERIC)
+		H.mind.adjust_skillrank(/datum/skill/combat/shields, 3, TRUE)	//May tone down to 2; seems OK.
+		// -- Start of section for god specific bonuses --
+		if(H.patron?.type == /datum/patron/divine/astrata)
+			H.mind.adjust_skillrank(/datum/skill/magic/holy, 1, TRUE)
+		if(H.patron?.type == /datum/patron/divine/dendor)
+			H.mind.adjust_skillrank(/datum/skill/labor/farming, 1, TRUE)
+			H.mind.adjust_skillrank(/datum/skill/misc/climbing, 1, TRUE)
+		if(H.patron?.type == /datum/patron/divine/noc)
+			H.mind.adjust_skillrank(/datum/skill/misc/reading, 3, TRUE) // Really good at reading... does this really do anything? No. BUT it's soulful.
+			H.mind.adjust_skillrank(/datum/skill/craft/alchemy, 1, TRUE)
+			H.mind.adjust_skillrank(/datum/skill/magic/arcane, 1, TRUE)
+		if(H.patron?.type == /datum/patron/divine/abyssor)
+			H.mind.adjust_skillrank(/datum/skill/labor/fishing, 2, TRUE)
+			ADD_TRAIT(H, TRAIT_WATERBREATHING, TRAIT_GENERIC)
+		if(H.patron?.type == /datum/patron/divine/necra)
+			H.mind.AddSpell(/obj/effect/proc_holder/spell/targeted/abrogation)
```

Comment:
this is given to templars by default now as it belongs to their cleric tier (T2)